xml/dns_versionbind.xml

```diff
@@ -0,0 +1,685 @@
+    <description>PowerDNS: Served by format without versionl</description>
+    <example>Served by PowerDNS - https://www.powerdns.com/</example>
+    <example>Served by PowerDNS - http://www.powerdns.com</example>
+    <param pos="0" name="service.vendor" value="PowerDNS"/>
+    <param pos="0" name="service.family" value="PowerDNS"/>
+  </fingerprint>
+  <fingerprint pattern="^Nominum Vantio(?: CacheServe)? ([\d.]+)$">
+    <description>Nominum Vantio CacheServe</description>
+    <example service.version="4.3.0.2">Nominum Vantio 4.3.0.2</example>
+    <example service.version="7.2.1.3">Nominum Vantio CacheServe 7.2.1.3</example>
+    <param pos="0" name="service.vendor" value="Nominum"/>
+    <param pos="0" name="service.family" value="Vantio"/>
+    <param pos="0" name="service.product" value="CacheServe"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Nominum Vantio ([\d.]+) \(build (\d+)\)$">
+    <description>Nominum Vantio CacheServe, with build</description>
+    <example service.version.version="114872">Nominum Vantio 5.4.5.1 (build 114872)</example>
+    <param pos="0" name="service.vendor" value="Nominum"/>
+    <param pos="0" name="service.family" value="Vantio"/>
+    <param pos="0" name="service.product" value="CacheServe"/>
+    <param pos="1" name="service.version"/>
+    <param pos="2" name="service.version.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Nominum ANS(?:Premier)? ([\d\.]+)$">
+    <description>Nominum Vantio AuthServ</description>
+    <example service.version="5.4.0.0">Nominum ANS 5.4.0.0</example>
+    <example service.version="5.4.0.0">Nominum ANSPremier 5.4.0.0</example>
+    <param pos="0" name="service.vendor" value="Nominum"/>
+    <param pos="0" name="service.family" value="Vantio"/>
+    <param pos="0" name="service.product" value="AuthServ"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^NSD ([\d.]*(?:b\d+)?)$">
+    <description>NLnet Labs Name Server Daemon</description>
+    <example service.version="3.2.18">NSD 3.2.18</example>
+    <example service.version="4.0.0b5">NSD 4.0.0b5</example>
+    <example service.version="4">NSD 4</example>
+    <example>NSD </example>
+    <param pos="0" name="service.vendor" value="NLnet Labs"/>
+    <param pos="0" name="service.family" value="NSD"/>
+    <param pos="0" name="service.product" value="dnsd"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^unbound ([\d.]+)$">
+    <description>NLnet Labs Unbound</description>
+    <example service.version="1.4.22">unbound 1.4.22</example>
+    <param pos="0" name="service.vendor" value="NLnet Labs"/>
+    <param pos="0" name="service.family" value="Unbound"/>
+    <param pos="0" name="service.product" value="unbound"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint flags="REG_ICASE" pattern="^unbound$">
```

Comment:
here and wherever else you are using `flags`, the preferred method is to write the regex such that it implements the equivalent of these flags but without the GNU.regex specific names.  In this case, `^(?i:unbound)$` would do it.